src/gui/properties/trackerlist.cpp

```diff
@@ -581,21 +575,36 @@ void TrackerList::saveSettings() const
     Preferences::instance()->setPropTrackerListState(header()->saveState());
 }
 
-QStringList TrackerList::headerLabels()
+QStringList TrackerList::headerLabels() const
 {
-    QStringList header;
-    header << "#";
-    header << tr("URL");
-    header << tr("Status");
-    header << tr("Received");
-    header << tr("Seeds");
-    header << tr("Peers");
-    header << tr("Downloaded");
-    header << tr("Message");
+    QStringList header {
+        "#"
+        , tr("URL")
+        , tr("Status")
+        , tr("Received")
+        , tr("Seeds")
+        , tr("Peers")
+        , tr("Downloaded")
+        , tr("Message")
+    };
 
     return header;
 }
 
+int TrackerList::visibleColumnsCount() const
+{
+    int visibleCols = 0;
+    for (unsigned int i = 0; i < COL_COUNT; ++i) {
+        if (!isColumnHidden(i))
+            ++visibleCols;
+
+        if (visibleCols > 1)
+            break;
```

Comment:
this doesn't match the function name, you should remove it or give the function a more precise name.